OSS-Docs-32: Removed instances of OCM

EricPonvelle opened the pull request:
This PR is based on the conversation in [OSSDOCS-23](https://issues.redhat.com/browse/OSSDOCS-23) (https://issues.redhat.com/browse/OSSDOCS-23) to remove the OCM acronym from our documentation.

For Reviewers:
- I used `OpenShift Cluster Manager` and `cluster manager` to replace OCM instances. 
- The only instances of OCM we should see currently are `{OCM}` for the variables and the structure
- I could use some input on if the OCM variable should be changed to `{cluster-manager}` but that seems less clear

JIRA: https://issues.redhat.com/browse/OSSDOCS-32

PREVIEW:

Because this touches so many topics, I'd request that anyone reviewing this ask for a preview that they wish to see.

1. https://deploy-preview-41055--osdocs.netlify.app/openshift-dedicated/latest/adding_service_cluster/adding-service.html
2. Pull secret phrasing - https://deploy-preview-41055--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-generate-aws-user-infra-install-config_installing-restricted-networks-aws

dayleparker commented on cloud_infrastructure_access/rosa-private-cluster.adoc:
```diff
@@ -7,7 +7,7 @@ toc::[]
 A {product-title} cluster can be made private so that internal applications can be hosted inside a corporate network. In addition, private clusters can be configured to have only internal API endpoints for increased security.
 
-// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager* (OCM).
+// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager*.
```
Please remove 'the' so it's consistent with how other docs refer to is as "OpenShift Cluster Manager"

dayleparker commented on modules/access-service.adoc:
```diff
@@ -6,18 +6,15 @@
 .Procedure
 
-. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OpenShift Cluster Manager (OCM)].
+. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OpenShift Cluster Manager].
```
This link should be https://console.redhat.com/openshift/ (the site changed in last year, so not sure how long redirects will continue working)

dayleparker commented on modules/adding-service-existing.adoc:
```diff
@@ -7,17 +7,17 @@
 * Your cluster meets all of the prerequisites for the service that you want to add on to your cluster.
 * For paid add-on services, note the following considerations:
-** If the organization has sufficient quota, and if the service is compatible with the cluster, the service appears in OCM.
+** If the organization has sufficient quota, and if the service is compatible with the cluster, the service appears in the cluster manager.
```
change to "in OpenShift Cluster Manager"

dayleparker commented on modules/adding-service-existing.adoc:
```diff
@@ -7,17 +7,17 @@
+** If the organization has sufficient quota, and if the service is compatible with the cluster, the service appears in the cluster manager.
 ** If the organization has never had quota, or if the cluster is not compatible, then the service does not display.
-** If the organization had quota in the past, but the quota is currently `0`, the service is still visible but disabled in OCM until you get more quota.
+** If the organization had quota in the past, but the quota is currently `0`, the service is still visible but disabled in the cluster manager until you get more quota.
```
Same as above - "OpenShift Cluster Manager"

dayleparker commented on modules/adding-service-existing.adoc:
```diff
@@ -27,7 +27,7 @@ To add a service to a cluster, you must be the cluster owner.
 .Procedure
 
-. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OCM].
+. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[the cluster manager].
```
Link needs correcting, and this should be called Openshift Cluster Manager.

dayleparker commented on modules/aws-cloudwatch.adoc:
```diff
@@ -6,7 +6,7 @@
 = Amazon CloudWatch
 
-Amazon CloudWatch forwards logs from {product-title} (ROSA) to the AWS console for viewing. You must first install the ROSA `cluster-logging-operator` using the `rosa` CLI before installing the Amazon CloudWatch service through the OpenShift Cluster Manager (OCM) console.
+Amazon CloudWatch forwards logs from {product-title} (ROSA) to the AWS console for viewing. You must first install the ROSA `cluster-logging-operator` using the `rosa` CLI before installing the Amazon CloudWatch service through the OpenShift Cluster Manager console.
```
I haven't seen it referred to as "OpenShift Cluster Manager console" so I think it would be more consistent (and searchable) as simply "through OpenShift Cluster Manager"

dayleparker commented on modules/cluster-telemetry.adoc:
```diff
@@ -69,8 +69,8 @@
+In {product-title} {product-version}, the Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://console.redhat.com/openshift[{console-redhat-com} (OpenShift Cluster Manager)].
 
-After you confirm that your {console-redhat-com} inventory is correct, either maintained automatically by Telemetry or manually by using OCM, link:https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift[use subscription watch] ...
+After you confirm that your {console-redhat-com} inventory is correct, either maintained automatically by Telemetry or manually by using the cluster manager, link:https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift[use sub ...
```
s/the cluster manager/OpenShift Cluster Manager

dayleparker commented on modules/deleting-service-cli.adoc:
```diff
@@ -6,13 +6,13 @@
 = Deleting a service from the CLI
 
-You can delete the add-on services from your {product-title} cluster through the OCM CLI.
+You can delete the add-on services from your {product-title} cluster through the OpenShift Cluster Manager CLI.
```
The ocm CLI isn't supported yet - this should be deleted from the doc.

dayleparker replied:
I can provide you background, I had to do the same in the OpenShift Cluster Manager doc.

dayleparker commented on modules/deleting-service-cli.adoc:
```diff
@@ -6,13 +6,13 @@
 .Procedure
 
-* To delete the add-on service from your cluster through the OCM CLI, enter the following command:
+* To delete the add-on service from your cluster through the cluster manager CLI, enter the following command:
```
Please delete this section as the ocm CLI is not supported.

dayleparker commented on modules/deleting-service.adoc:
```diff
@@ -3,7 +3,7 @@
 
 [id="deleting-service_{context}"]
-= Deleting a service from OCM
+= Deleting a service from the cluster manager
```
s/the cluster manager/OpenShift Cluster Manager

dayleparker commented on modules/notification-subscribe.adoc:
```diff
@@ -11,7 +11,7 @@ You can add notification contacts for your {product-title} cluster. When an even
 .Procedure
 
-. From link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)], navigate to the *Clusters* page and select your cluster.
+. From link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager], navigate to the *Clusters* page and select your cluster.
```
Link needs correcting to console.redhat.com

dayleparker commented on modules/ocm-accessing.adoc:
```diff
@@ -3,9 +3,9 @@
+= Accessing the Red Hat OpenShift Cluster Manager
 
-You can access OCM with your configured OpenShift account.
+You can access the cluster manager with your configured OpenShift account.
```
s/the cluster manager/OpenShift Cluster Manager

dayleparker commented on modules/ocm-accessing.adoc:
```diff
@@ -3,9 +3,9 @@
 
 [id="accessing-ocm_{context}"]
-= Accessing the Red Hat OpenShift Cluster Manager (OCM)
+= Accessing the Red Hat OpenShift Cluster Manager
```
Delete "the"

dayleparker commented on modules/ocm-accessing.adoc:
```diff
@@ -14,4 +14,4 @@ You can access OCM with your configured OpenShift account.
 .Procedure
 
-* Log in to the link:https://console.redhat.com/openshift[OCM web console] using your login credentials.
+* Log in to the link:https://console.redhat.com/openshift[cluster manager web console] using your login credentials.
```
s/the cluster manager/OpenShift Cluster Manager

EricPonvelle commented:
@dayleparker - this is a lot. I've done my best to correct the following items:

1. All instances of `OCM` have been removed.
2. Removed most instances of `cluster manager` except where it referred to an actual person.
3. Fixed the URLs to be mostly be a variable so that we only change the URL in one location.
4. Corrected the headers.
5. Fixed some instances of hard returns

I am certain I've missed some things or may have been over-zealous in my replacements, so can you assess this to the best of your time constraints? I'll request Peer Review to help test this.

dayleparker commented on modules/common-attributes.adoc:
```diff
@@ -23,7 +23,10 @@ endif::[]
+:cluster-manager-first: Red Hat OpenShift Cluster Manager
+:cluster-manager: OpenShift Cluster Manager
+:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager]
+:cluster-manager-url-pull: link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site]
```
For the text that is visible in the doc, does it currently read "pull secret from the Red Hat OpenShift Cluster Manager site"?

If yes, this should be changed to "pull secret from Red Hat OpenShift Cluster Manager". We don't refer to it as a 'site' anywhere.

dayleparker commented on modules/config-aws-access.adoc:
```diff
@@ -55,7 +55,7 @@ It is not recommended to set a permissions boundary.
 .. Gather the IAM user’s Amazon Resource Name (ARN). The ARN will have the following format: `arn:aws:iam::000111222333:user/username`. Click *Close*.
 
-. Open the {console-redhat-com} in your browser and select the cluster you want to allow AWS infrastructure access.
+. Open the {cluster-manager-url} in your browser and select the cluster you want to allow AWS infrastructure access.
```
Remove "the", or might be better changed to "Navigate to OpenShift Cluster Manager" or "In OpenShift Cluster Manager, select the cluster...."

dayleparker commented on modules/attributes-openshift-dedicated.adoc:
```diff
@@ -5,8 +5,10 @@
+:cluster-manager-first: Red Hat OpenShift Cluster Manager
+:cluster-manager: OpenShift Cluster Manager
+:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager]
+:cluster-manager-url-pull: link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site]
```
Change "pull secret from the Red Hat OpenShift Cluster Manager site" to "pull secret from Red Hat OpenShift Cluster Manager"

dayleparker commented on modules/dedicated-managing-dedicated-administrators.adoc:
```diff
@@ -5,9 +5,7 @@
-Administrator roles are managed using a `dedicated-admins` group on the cluster.
-Existing members of this group can edit membership via the
-link:https://console.redhat.com/openshift[{console-redhat-com}] site.
+Administrator roles are managed using a `dedicated-admins` group on the cluster. Existing members of this group can edit membership via the {cluster-manager-url}.
```
Remove "the"

dayleparker commented on modules/deleting-cluster.adoc:
```diff
@@ -8,11 +8,11 @@
 
 
-You can delete your {product-title} cluster in the {OCM}.
+You can delete your {product-title} cluster in the {cluster-manager-first}.
```
remove "the"

dayleparker commented on modules/deleting-service.adoc:
```diff
@@ -3,13 +3,13 @@
+= Deleting a service from {cluster-manager-first}
 
-You can delete the add-on services from your {product-title} cluster through the {OCM}.
+You can delete the add-on services from your {product-title} cluster through the {cluster-manager}.
```
Remove 'the'

dayleparker commented on modules/displaying-potential-issues-with-your-cluster.adoc:
```diff
@@ -5,15 +5,15 @@
 = Displaying potential issues with your cluster
 
-This section describes how to display the Insights report in the {console-redhat-com}.
+This section describes how to display the Insights report in the {cluster-manager-url}.
```
Remove 'the'

dayleparker commented on modules/displaying-potential-issues-with-your-cluster.adoc:
```diff
@@ -5,15 +5,15 @@
 .Prerequisites
 
-* Your cluster is registered in the {console-redhat-com}.
+* Your cluster is registered in the {cluster-manager-url}.
```
Remove 'the' before the attribute.

dayleparker commented on modules/displaying-the-insights-status-in-the-web-console.adoc:
```diff
@@ -5,11 +5,11 @@
 = Displaying the Insights status in the web console
 
-Insights repeatedly analyzes your cluster and you can display the status of identified potential issues of your cluster in the {product-title} web console. This status shows the number of issues in the different categories and, for further details, links to the reports in the {console-redhat-com}.
+Insights repeatedly analyzes your cluster and you can display the status of identified potential issues of your cluster in the {product-title} web console. This status shows the number of issues in the different categories and, for further details, links to the reports in the {cluster-manager-url}.
```
Remove 'the' before OpenShift Cluster Manager.

dayleparker commented on modules/installation-generate-aws-user-infra-install-config.adoc:
```diff
@@ -61,8 +61,7 @@ The AWS access key ID and secret access key are stored in `~/.aws/credentials` i
 ... Enter a descriptive name for your cluster.
-... Paste the pull secret that you obtained from the
-link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site.
+... Paste the pull secret that you obtained from the {cluster-manager-url-pull}.
```
I'm not sure this sentence makes sense with the attribute, please check.

dayleparker commented:
Hey @EricPonvelle!

I've gone through the changes and it looks good from what I can tell - I can confirm I don't see any more "OCM" acronyms. I'm sure the attributes will save writers lots of work in the future :)

Things to address:

- I attempted to read the raw Asciidoc with the attributes, and I found some places where there's a lingering 'the' or 'site' that should be fixed. I'm not sure if I flagged it in all the attribute files but I tried (there's so many of them!). This is so that we're always calling it (between our docs, and in the product pages, etc) 'OpenShift Cluster Manager' rather than 'the OpenShift Cluster Manager site', etc.  

- Somewhere in your OpenShift Cluster Manager guide, we need a link to the full OCM doc that our team maintains still so customers have an easier time accessing that when they're in the OpenShift docs. Is this being addressed in another Jira? Here's the link: https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index

Thanks for your thoroughness doing all these edits!
Cheers,
Dayle

jeana-redhat commented on modules/access-cluster.adoc:
```diff
@@ -7,7 +7,7 @@
+After you have configured your identity providers, users can access the cluster from {cluster-manager-first}.
 
 .Prerequisites
 
```
There's an {OCM} on line 14 that is showing in plaintext in preview

bmcelvee replied:
This is still showing {OCM} and I'm not sure why

jeana-redhat commented on modules/access-service.adoc:
```diff
@@ -7,18 +7,15 @@
 .Procedure
 
-. Navigate to the *Clusters* page in link:https://cloud.redhat.com/openshift/[OpenShift Cluster Manager (OCM)].
+. Navigate to the *Clusters* page in {console-red-hat}.
```
This attribute is rendering in plain text in the preview.

jeana-redhat reviewed (commented):
Submitting a batch where I'm at, but there are still some odd things going on with files and attributes. Should probably look into that before proceeding with further review.

bmcelvee commented on cicd/pipelines/installing-pipelines.adoc:
```diff
@@ -19,7 +19,7 @@ This guide walks cluster administrators through the process of installing the {p
 
 ifdef::openshift-origin[]
-* Ensure that you have downloaded the link:https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in the xref:../../installing/installing_gcp/installing-gcp-customizations.adoc#installation-obtaining-installer_installing-gcp-customizations[Obtaining the installation program] to install this Operator.
+* Ensure that you have downloaded the {cluster-manager-url-pull} as shown in the xref:../../installing/installing_gcp/installing-gcp-customizations.adoc#installation-obtaining-installer_installing-gcp-customizations[Obtaining the installation program] to install this Operator.
```
Looks like Pipelines isn't currently included in Origin (OKD), so this should be fine.

bmcelvee commented on cloud_infrastructure_access/dedicated-aws-access.adoc:
```diff
@@ -63,7 +63,7 @@ It is not recommended to set a permissions boundary.
 
 [id=dedicated-aws-ocm-iam-role]
-=== Granting the IAM role from the OpenShift Cluster Manager
+=== Granting the IAM role from {cluster-manager-first}
```
I don't see this assembly included in any of our topic maps. It might be worth verifying that this one wasn't missed in the migration.

bmcelvee commented:
I tried to go check as much as possible, but ultimately the spot checks looked pretty good. There are a few oddities, which are all commented on. Good work, though - this is a huge effort!

dayleparker commented on modules/images-update-global-pull-secret.adoc:
```diff
@@ -14,9 +14,9 @@ You can update the global pull secret for your cluster by either replacing the c
+To transfer your cluster to another owner, you must first initiate the transfer in {cluster-manager-url}, and then update the pull secret on the cluster. Updating a cluster's pull secret without initiating the transfer in {cluster-manager} causes the cluster to stop reporting Telemetry metrics in {cluster-manager}.
 
-For more information link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[about transferring cluster ownership], see "Transferring cluster ownership" in the Red Hat OpenShift Cluster Manager documentation. 
+For more information link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[about transferring cluster ownership], see "Transferring cluster ownership" in the {cluster-manager-first} documentation.
```
Hi Eric - if it's not too late, the OCM doc link is now updated to 2022 - can that be changed still in the PR? Let me know if you need any additional details, this is the link:

https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index

EricPonvelle commented:
/cherrypick enterprise-4.9

EricPonvelle commented:
/cherrypick enterprise-4.10

bergerhoffer commented:
/cherrypick enterprise-4.10
